### ISSUE TITLE
Packaging the project as an executable

### ISSUE THREAD

**Issue by ivaneidel** [3rdIteration/btcrecover#230]
Hi there!

First of all, great project! Been using it for a few days and it has proven to be really helpful.

I'm a soft developer (specifically, I work developing a non-custodial wallet) and a user of us wrongly backed up his seeds... We've been trying to help him recover them without him revealing his seeds to us when we came across this tool.

The question: We want our user to try to recover the words himself using 'seedrecover' but he lacks the technical knowleadge for installing python and running commands in the command line. We've been trying to package the project for Windows to give him an executable that he can run with a single click, but neither 'py2exe' nor 'pyinstaller' seem to succeed in doing the job.

Any chance you can package it for us or give us a hand or point us in the right direction...?

We know it's a lot to ask, but it would be awesome for us to have this tool in our toolbelt for users with this problem to try n solve it themselves.

Once again, awesome project!

Best regards!

**Comment by 3rdIteration** [comment]
Have you directed them to the video where I step through how to do this in Windows? (It's much simpler than back in the python 2 version)

I was planning to go executables but I'm not sure I really want to go down the path of signing releases, etc.

**Comment by 3rdIteration** [comment]
I should also add that if it's just an error on transcribing the seed, you don't need any additional command line arguments for it. Once the requirements are installed, you can double click on seedrecover.py and the GUI is all you need.

Would a simple batch script to install the requirements help? (So you could just do it all via the windows GUI)

**Comment by ivaneidel** [comment]
Hi there, thanks for the fast reply!!

We have seen your videos, we'll take a look at them again. Anyway, this user doesn't speak English, so maybe we'll need to make  a video in Spanish :shrug: .

We had tested the GUI but couldn't get it to work. We later found out we were making a mistake in the steps. We have tried it again and it's working like a charm.

We hadn't thought of a batch script... It would be a great option... Could we bother you with it? It would be to install python and the dependencies of the 'seedrecover'.

Thank you very much in advance!

We're looking forward to hearing from you.

Cheers!
